### PR TITLE
Added 'FILES_SAVE' for self tests of Robot Framework

### PR DESCRIPTION
**Pull request by HolQue**
('log_level' test) in TestTrigger configuration

### CODE REVIEW

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi Holger,
looks good to me!
release_info?
Thank you,
 Thomas

**Comment by HolQue** [comment]
Hi Thomas,

good question. I am not sure. The added information is for the build process, so that the build process knows where to find additional log files. Expected impact is that also the log files of the "log level" self test are part of the downloadable log archive. Are readers of a release mail really interested in such kind of details?

Hmm, I will mention that a new self test is part of the build tool chain. This might be interesting.

**Comment by HolQue** [comment]
Done

File: config/robotframework_aio/release_items_Robotframework_AIO.json
Comment (test-fullautomation) [inline config/robotframework_aio/release_items_Robotframework_AIO.json:1]: Hi Holger,
jupp, thats the end user perspective.
Thank you,
 Thomas

**Review by test-fullautomation — APPROVED** [review]

**Pull request merged**